src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -285,6 +285,23 @@
     <Compile Include="System\ExitCodeTests.Unix.cs" />
   </ItemGroup>
 
+  <!--
+    These tests are currently not R2R-Compatible, so we disable them whenever
+    we are using the TestReadyToRun flag.
+  -->
+  <ItemGroup Condition="'$(TestReadyToRun)' == 'true'">
+    <Compile Remove="System\Runtime\JitInfoTests.cs" />
+    <Compile Remove="System\Type\TypeTests.Get.cs" />
+  </ItemGroup>
+
+  <!--
+    This test surprisingly only fails on Linux and Windows when TestReadyToRun
+    is present. OSX runs just fine.
+  -->
+  <ItemGroup Condition="'$(TestReadyToRun)' == 'true' and ('$(TargetPlatformIdentifier)' == 'linux' or '$(TargetPlatformIdentifier)' == 'windows')">
+    <Compile Remove="System\EnumTests.cs" />
+  </ItemGroup>
```

Comment:
Do those tests fail to compile or successfully run? If the latter, do we have a way to determine at run time if we are testing crossgend bits?